src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class CalculatorTest {
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+    private static final double delta = 0.0;
```

Comment:
```suggestion
    private static final double DELTA = 0.0;
```
constant naming examples: SELECTOR, BASE_URL, SOME_CONSTANT_NAME